sagemaker-clarify/bias-and-explainability/student_performance_clarify.ipynb

```diff
@@ -0,0 +1,858 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    !wget -O student.zip http://archive.ics.uci.edu/ml/machine-learning-databases/00320/student.zip
</pre><p>I have made the dataset available at <code>s3://sagemaker-sample-files/datasets/tabular/uci_student_performance/student-mat.csv</code>.  Please use that version instead and download with boto3 rather than wget. I already removed the school column from this version.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2916/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>